test/uk/gov/hmrc/merchandiseinbaggage/content/CheckYourAnswerAmendImportContentSpec.scala

```diff
@@ -25,12 +25,14 @@ import uk.gov.hmrc.merchandiseinbaggage.model.api.YesNo.Yes
 import uk.gov.hmrc.merchandiseinbaggage.model.api.calculation.{CalculationResult, ThresholdCheck, WithinThreshold}
 import uk.gov.hmrc.merchandiseinbaggage.model.core.DeclarationJourney
 import uk.gov.hmrc.merchandiseinbaggage.smoketests.pages.CheckYourAnswersPage
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub._
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 
 import scala.jdk.CollectionConverters._
 
 class CheckYourAnswerAmendImportContentSpec extends CheckYourAnswersPage with CoreTestData {
 
+  private val mibStub = app.injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these changes, thanks